src/main/java/ru/sbtqa/tag/pagefactory/support/MobileConsole.java

```diff
@@ -0,0 +1,42 @@
+package ru.sbtqa.tag.pagefactory.support;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.sbtqa.tag.pagefactory.drivers.TagMobileDriver;
+
+public class MobileConsole {
```

Comment:
AdbConsole. Because when we will add Iphone it will confuse everything. Also you can describe the interface which will abstract you from mobile platform.